runtime/jcl/common/sun_reflect_ConstantPool.c

```diff
@@ -459,6 +459,41 @@ Java_sun_reflect_ConstantPool_getFieldAtIfLoaded0(JNIEnv *env, jobject unusedObj
 }
 
 jobject JNICALL
+Java_java_lang_invoke_MethodHandle_getClassNameAt(JNIEnv *env, jobject unusedObject, jobject constantPoolOop, jint cpIndex)
```

Comment:
Please a function comment that describes the arguments, the return types and any invariants.

Reply:
fixed